### PR TITLE
WDS-125 loading and options type

### PR DESCRIPTION
**Pull request by Ctepan**
add prop to select options type
add loading state
add ellipsis on select text overflow
add stories for new features

### CODE REVIEW

File: packages/ui/src/components/Select/SSelectOption.vue [hunk 1]
```diff
@@ -28,30 +40,29 @@ const emit = defineEmits<(event: 'toggle') => void>()
...
-      <div class="s-select-option__right-check-wrapper">
-        <template v-if="selected">
-          <IconCheckMark />
-        </template>
-      </div>
+    <template v-if="type === SelectOptionType.Default">
+      <IconCheckMark
+        v-if="selected"
+        :width="CHECK_ICON_SIZE[api.size]"
+        :height="CHECK_ICON_SIZE[api.size]"
```
[the remark this answers is not in the document]
Comment (Ctepan) [reply packages/ui/src/components/Select/SSelectOption.vue:64]: I don't think they rely on em

[the remark this answers is not in the document]
Comment (0x009922) [reply packages/ui/src/components/Select/SSelectOption.vue:64]: Okay, that's my misunderstanding - I am used to how it works in icones.js

File: .changeset/silver-countries-help.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@soramitsu-ui/ui': minor
+---
+
+**feat**(`SSelect`): add prop to select options type
```
[the remark this answers is not in the document]
Comment (Ctepan) [reply .changeset/silver-countries-help.md]: fixed

File: .changeset/light-toes-remember.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+'@soramitsu-ui/ui': minor
+---
+
+**feat**(`SSelect`): add loading state
```
[the remark this answers is not in the document]
Comment (Ctepan) [reply .changeset/light-toes-remember.md]: fixed

**Review by stefashkaa — APPROVED** [review]

**Review by 0x009922 — APPROVED** [review]

**Pull request merged**